castle/cms/browser/controlpanel/announcements.py

```diff
@@ -32,6 +32,10 @@
 reg_key = 'castle.subscriber_categories'
 
 
+def toUnicode(txt):
```

Comment:
Maybe you want to use `from Products.CMFPlone.utils import safe_unicode` instead?

Reply:
will do